internal/services/storage/storage_container_resource_test.go

```diff
@@ -183,6 +185,133 @@ func TestAccStorageContainer_web(t *testing.T) {
 	})
 }
 
+func TestAccStorageContainer_mgmtBasic(t *testing.T) {
```

Comment:
we can call these `management` or even `resourceManager` - since `management` is mostly an MS-internal term in this context tbh